docs/system_requirements/index.sdoc

```diff
@@ -241,19 +241,31 @@ As a Zephyr RTOS user, I want to be able to give my threads different priorities
 
 [/SECTION]
 
+[SECTION]
+TITLE: Timers
+
 [REQUIREMENT]
 UID: ZEP-SYRS-18
 STATUS: Draft
-TYPE: High Level
+TYPE: Non-Functional
 COMPONENT: Timers
-TITLE: Timers
+TITLE: Time based events
 STATEMENT: >>>
 The Zephyr RTOS shall provide a framework for managing time-based events.
 <<<
-USER_STORY: >>>
-As a Zephyr RTOS user, I want to start, suspend, resume and stop timers which shall trigger an event on a set expiration time.
+
+[REQUIREMENT]
+UID: ZEP-SYRS-20
+STATUS: Draft
+TYPE: Non-Functional
+COMPONENT: Timers
+TITLE: Timers time base
+STATEMENT: >>>
+The Timers of the Zephyr RTOS shall use the kernel's system clock to measure time.
```

Comment:
This is a design decision. Just not clear why a timer implementation shall be restricted to the system clock. A justification for this descision would be helpful.

Reply:
> This is a design decision. Just not clear why a timer implementation shall be restricted to the system clock. A justification for this descision would be helpful.

Ok so removing the requirement here would be the best way? And the justification should be done in the design itself your should we do it on this level?